lib/_http_server.js

```diff
@@ -146,12 +155,24 @@ function ServerResponse(req) {
     this.useChunkedEncodingByDefault = chunkExpression.test(req.headers.te);
     this.shouldKeepAlive = false;
   }
+
+  const httpObserverCount = observerCounts[IDX_PERFORMANCE_ENTRYTYPE_HTTP];
+  if (httpObserverCount !== undefined && httpObserverCount > 0) {
```

Comment:
If `httpObserverCount` is `undefined` the second check will be falsy. It seems a bit redundant therefore?

```suggestion
  if (httpObserverCount > 0) {
```

Reply:
Indeed, i've removed the first check